src/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rdoc', '~>4.0'
+gem 'rspec', '~> 3.0'
```

Comment:
Don't you want to move this gem dependencies to your gemspec file? Then Gemfile will be:
```
source 'https://rubygems.org'

# Specify your gem's dependencies in symengine.gemspec
gemspec
```

And in your gemspec you'll be able to differ dependencies into development and run-time, see http://guides.rubygems.org/specification-reference/#add_development_dependency .